docs/troubleshooting-guide/bugs.md

```diff
@@ -1,47 +1,33 @@
 If you come across something that looks like a bug, we suggest collecting the following information to help us reproduce the issue.
 
-1. Server logs
+1. Metabase server logs
 2. Javascript console logs
 3. Can it be reproduced on the sample dataset?
 4. Your Metabase version
-5. Where Metabase is running (Docker image, AWS Elastic Beanstalk, etc)
-6. What browser version
+5. Where Metabase is running (Docker image, AWS Elastic Beanstalk, Heroku, Linux/Ubuntu, etc)
+6. Which database is used
+7. What browser version
 
 ## Helpful tidbits
 
 ### Accessing the Metabase server logs
-While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the drop down menu in the upper right hand corner.
+While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the gear drop down menu in the upper right hand corner -> Admin -> Troubleshooting -> Logs.
 
-![Logs Screenshot](images/ServerLogs.png)
```

Comment:
I'm not sure if I should delete this file or if it would have consequences for earlier versions?

Reply:
Should be safe to delete — we version the docs, so past versions should still have a copy of this image in their built doc directories (e.g. here https://metabase.com/docs/v0.28.0/troubleshooting-guide/images/ServerLogs.png).